libraries/AP_NavEKF/AP_Nav_Common.cpp

```diff
@@ -6,13 +6,16 @@
 /*
   write an EKF timing message
  */
-void Log_EKF_Timing(const char *name, uint64_t time_us, const struct ekf_timing &timing)
+void Log_EKF_Timing(const char * name, const uint8_t core, uint64_t time_us, const struct ekf_timing &timing)
 {
     AP::logger().Write(
         name,
-        "TimeUS,Cnt,IMUMin,IMUMax,EKFMin,EKFMax,AngMin,AngMax,VMin,VMax",
-        "QIffffffff",
+        "TimeUS,C,Cnt,IMUMin,IMUMax,EKFMin,EKFMax,AngMin,AngMax,VMin,VMax",
```

Comment:
I guess we couldn't fit "Core" or "Instance" in the title instead of just "C"?

Reply:
There's a 64 char limit, which this is exactly at with `C`. So if we can remove 3 letters elsewhere then we can have `Core`....